src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -72,53 +72,45 @@ public Locomotive getLocomotive() {
     public Locomotive() {
     }
 
-    /**
-     * Constructor for Unit Tests
-     */
-    public Locomotive(ConductorConfig configuration, AppiumDriver driver) {
```

Comment:
Seems pretty safe to remove this. It should be only used for the unit tests in this project.